src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -133,14 +137,26 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
 
     if (!asset || !underlyingAsset || !marketData) continue
 
+    const apy = bnOrZero(
+      midgardPools.find(pool => pool.asset === thorchainPool.asset)?.saversAPR,
+    ).toString()
     const tvl = bnOrZero(thorchainPool.savers_units)
       .div(bn(10).pow(THOR_PRECISION))
       .times(marketData.price)
       .toFixed()
+    // NOT the same as the TVL:
+    // - TVL is the fiat total of assets locked
+    // - supply is the fiat total of assets locked, including the accrued value, accounting in the max. cap
+    const saversSupplyIncludeAccruedFiat = bnOrZero(thorchainPool.savers_depth)
+      .div(bn(10).pow(THOR_PRECISION))
+      .times(marketData.price)
+      .toFixed()
+    const saversMaxSupplyFiat = bnOrZero(thorchainPool.synth_supply)
+      .plus(thorchainPool.synth_supply_remaining)
+      .div(bn(10).pow(THOR_PRECISION))
+      .times(marketData.price)
+      .toFixed()
```

Comment:
Not consumed yet - didn't test whether or not the numbers are correct but needed at type-level now, will revisit when wiring up